### PR TITLE
chore(neuron-ui): add settings ui

### PR DESCRIPTION
**Pull request by duanyytop**

### CODE REVIEW

File: packages/neuron-ui/src/components/Settings/index.tsx [hunk 1]
```diff
@@ -1,3 +1,159 @@
...
+  ]
+  const networks: string[] = ['Mainnet', 'Testnet']
+  const [networkSelected, setNetworkSelected] = useState(networks[0])
+  const [walletSelected, setWalletSelected] = useState(wallets[0])
+  const [walletsState, setWalletsState] = useState(wallets)
+
+  const deleteWallet = (walletName: string) => {
+    const index = wallets.indexOf(walletName)
+    if (index > -1) {
+      wallets.splice(index, 1)
```
Comment (Keith-CY) [inline packages/neuron-ui/src/components/Settings/index.tsx]: Usually, we don't change source data.

Leave it and refactor later.

File: packages/neuron-ui/src/components/Settings/index.tsx [hunk 2]
```diff
@@ -1,3 +1,159 @@
...
+    }
+    setWalletSelected(wallets[0])
```
Comment (Keith-CY) [inline packages/neuron-ui/src/components/Settings/index.tsx]: Why will deleting wallet set `wallet selected`?

File: packages/neuron-ui/src/components/Settings/index.tsx [hunk 3]
```diff
@@ -1,3 +1,159 @@
...
+    setWalletsState(wallets)
+  }
+
+  const addWallet = () => {
+    wallets.push(`Wallet Name${wallets.length + 1}`)
+    setWalletSelected(wallets[0])
```
Comment (Keith-CY) [inline packages/neuron-ui/src/components/Settings/index.tsx]: Why will adding wallet set `selected wallet`

File: packages/neuron-ui/src/components/Settings/index.tsx [hunk 4]
```diff
@@ -1,3 +1,159 @@
...
+              <AddWallet
+                onClick={() => {
+                  addWallet()
+                }}
+              >
+                Add
+              </AddWallet>
+              <DeleteWallet
+                onClick={() => {
+                  deleteWallet(walletSelected)
```
Comment (Keith-CY) [inline packages/neuron-ui/src/components/Settings/index.tsx]: `walletSelected` is the state, it could be referenced by the state directly.

**Comment by duanyytop** [comment]
> I'd love to see these three parts of their own components.
> 
> The Settings/Preferences panel should be a simple component that holds the tab view and take care of the UI states, while `/settings` (or `/settings/general`), `/settings/network`, `/settings/wallets` route and go to separate components.

OK. I guess you mean to take `settings/general`, `/settings/network`, `/settings/wallets` as separate components , not separate pages ?

File: packages/neuron-ui/src/components/Settings/index.tsx (hunk 1 above)
Comment (duanyytop) [reply packages/neuron-ui/src/components/Settings/index.tsx]: Oh, I can not upload latest code. I'm sorry.

File: packages/neuron-ui/src/components/Settings/index.tsx (hunk 2 above)
Comment (duanyytop) [reply packages/neuron-ui/src/components/Settings/index.tsx]: Oh, I did not upload latest code. I'm sorry.

File: packages/neuron-ui/src/components/Settings/index.tsx (hunk 3 above)
Comment (duanyytop) [reply packages/neuron-ui/src/components/Settings/index.tsx]: Oh, I did not upload latest code. I'm sorry.

File: packages/neuron-ui/src/components/Settings/index.tsx (hunk 4 above)
Comment (duanyytop) [reply packages/neuron-ui/src/components/Settings/index.tsx]: OK, I get it.

**Comment by ashchan** [comment]
> > I'd love to see these three parts of their own components.
> > The Settings/Preferences panel should be a simple component that holds the tab view and take care of the UI states, while `/settings` (or `/settings/general`), `/settings/network`, `/settings/wallets` route and go to separate components.
> 
> OK. I guess you mean to take `settings/general`, `/settings/network`, `/settings/wallets` as separate components , not separate pages ?

Separate components, yes!

Separate pages, depends. But if we link from anywhere in the app to one of them, we expect to see that tab together with other tabs (titles).

**Comment by duanyytop** [comment]
I have refactor settings code and now it consist of three separate components .  I fetched the latest code of develop and rebase to my branch settings_ui.

File: packages/neuron-ui/src/components/Settings/TabBar.tsx [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+  width: 200px;
+  height: 40px;
+  border-radius: 10px;
+  text-align: center;
+  line-height: 60px;
+  color: black;
+`
+const TabBar = (props: any) => {
+  return (
+    <TabBarPenal>
```
Comment (ashchan) [inline packages/neuron-ui/src/components/Settings/TabBar.tsx]: Is `Penal` a typo?

File: packages/neuron-ui/src/components/Settings/general.tsx [hunk 6]
```diff
@@ -0,0 +1,28 @@
+import React from 'react'
```
Comment (ashchan) [inline packages/neuron-ui/src/components/Settings/general.tsx]: `general.tsx` file name should be capitalized. So is `wallets`.

File: packages/neuron-ui/src/components/Settings/TabBar.tsx (hunk 5 above)
Comment (duanyytop) [reply packages/neuron-ui/src/components/Settings/TabBar.tsx]: It should be panel. I'm sorry.

File: packages/neuron-ui/src/components/Settings/general.tsx (hunk 6 above)
Comment (duanyytop) [reply packages/neuron-ui/src/components/Settings/general.tsx]: OK. It is my fault.

**Pull request merged**